.github/workflows/gh-build-and-test.yml

```diff
@@ -300,9 +300,9 @@ jobs:
       - name: Run cuda.core tests
         shell: bash --noprofile --norc -xeuo pipefail {0}
         run: |
-          if [[ $SKIP_CUDA_BINDINGS_TEST == 1 ]]; then
+          if [[ ${{ matrix.python-version }} == "3.13" ]]; then
```

Comment:
Just curious, what was the motivation for switching the order of `if`s around here?